src/geode/mesh/CMakeLists.txt

```diff
@@ -66,6 +68,7 @@ add_geode_library(
         "core/register_mesh.cpp"
         "core/regular_grid.cpp"
         "core/solid_mesh.cpp"
+        "core/surface_edges.cpp"
         "core/surface_mesh.cpp"
         "core/tetrahedral_solid.cpp"
```

Comment:
Is `core/detail/edge_view_impl.h` to be removed?

Reply:
it was'nt added... no comment